### PR TITLE
docs: rewrite local development section

### PR DESCRIPTION
**Pull request by TC-MO**
rewrite the local development section for clarity & conciseness restructure the document with additional headings
add admonitions about:
prequisites
monetization
benefits of local development

### CODE REVIEW

File: sources/platform/actors/development/quick_start/images/actor-create.gif
Comment (jbartadev) [inline sources/platform/actors/development/quick_start/images/actor-create.gif:1]: The template installation takes quite a lot of time in the new gif 😄

File: sources/platform/actors/development/quick_start/start_locally.md [hunk 1]
```diff
@@ -1,89 +1,131 @@
...
 ```bash
 apify create
 ```
 
-You will be prompted to name your Actor (e.g, my-actor), and choose the programming language you would like to use (JavaScript, TypeScript, or Python). Next, you will be presented with a list of Actor development templates. After choosing your preferred template the CLI will:
+The CLI will prompt you to:
 
-- Create a `my-actor` directory containing boilerplate code.
-- Install all the project dependencies.
+1. _Name your Actor_: Enter a descriptive name for your actor, such as `my-actor`
```
Comment (jbartadev) [inline sources/platform/actors/development/quick_start/start_locally.md:48]: > Enter a descriptive name for your actor, such as `my-actor`

The `my-actor` is not very descriptive 😄 maybe we can come with with something a bit better?

File: sources/platform/actors/development/quick_start/start_locally.md [hunk 2]
```diff
@@ -1,89 +1,131 @@
...
 
-![Actor source code](./images/actor-local-code.png)
+- `srx/main.js`: This file contains the actual code of your Actor
 
-> The following chapters will teach you more about an Actor's structure and files.
+### `actor` Directory
 
-## 4. Run it locally
+- `actor/json`: This file diefines the Actor's configuration, such as input and output specifications.
+- `Dockerfile`: This file contains instructions for building the Docker image for your Actor.
```
Comment (jbartadev) [inline sources/platform/actors/development/quick_start/start_locally.md:76]: How about improving the linking here too? (I hard-coded the links there for review purposes)

```suggestion
- `[actor/json](https://docs.apify.com/platform/actors/development/actor-definition/actor-json)`: This file defines the Actor's configuration, such as input and output specifications.
- `[Dockerfile](https://docs.apify.com/platform/actors/development/actor-definition/dockerfile)`: This file contains instructions for building the Docker image for your Actor.
```

BTW, I noticed there is a typo (`diefines` - `defines`). How is it possible that it wasn't picked up by a spell-check extension? 👀

**Review by jbartadev — COMMENTED** [review]
Nice improvements!

File: sources/platform/actors/development/quick_start/images/actor-create.gif
Comment (TC-MO) [reply sources/platform/actors/development/quick_start/images/actor-create.gif:1]: Yeah, I'm not sure ,whether it's my machine or what, I'll try to cut it a bit so it takes way less time for viewer to see whole process

File: sources/platform/actors/development/quick_start/start_locally.md (hunk 2 above)
Comment (TC-MO) [reply sources/platform/actors/development/quick_start/start_locally.md:76]: good catch, let me investigate around in Vale and my own setup.

**Pull request merged**